dspy/adapters/chat_adapter.py

```diff
@@ -115,86 +115,6 @@ def format_fields(self, signature, values, role):
         return format_fields(fields_with_values)
 
 
-def format_blob(blob):
```

Comment:
Moved these utilities into a shared `adapters/utils.py` module, since they're identical but repeated across JSONAdapter and ChatAdapter.

(Except that JSONAdapter doesn't supported images yet and throws an "unsupported" exception. This behavior is preserved by wrapping `utils.format_field_value()` with a`if field_info.annotation is Image: throw` block